### PR TITLE
open menu by native js

### PR DESCRIPTION
**Pull request by lory87**
**This pull request fixes the following issues**:

- it's not possible to work with jquery here
- replaced hover with click event

### CODE REVIEW

File: djangocms_admin_style/static/js/base-admin.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+function toggleClass (element, className) {
+
+    var classString = element.className, nameIndex = classString.indexOf(className);
+
+    if (!element || !className) {
+        return true;
+    }
+
+    if (nameIndex == -1) {
```
Comment (sergii-kulakovskyi) [inline djangocms_admin_style/static/js/base-admin.js]: === instead of ==

File: djangocms_admin_style/static/js/base-admin.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+function toggleClass (element, className) {
+
+    var classString = element.className, nameIndex = classString.indexOf(className);
```
Comment (dhorelik) [inline djangocms_admin_style/static/js/base-admin.js]: `var nameIndex = ..` - new line

File: djangocms_admin_style/static/js/base-admin.js (hunk 1 above)
Comment (dhorelik) [inline djangocms_admin_style/static/js/base-admin.js]: `===`

File: djangocms_admin_style/sass/components/_header.scss [hunk 3]
```diff
@@ -110,6 +109,10 @@
                             font-weight: bold;
                             font-size: $font-size-normal;
                         }
+                        &:hover {
+                            color: $white!important;
```
Comment (sergii-kulakovskyi) [inline djangocms_admin_style/sass/components/_header.scss]: I think better to add space between `$white` & `!important`

File: djangocms_admin_style/static/js/base-admin.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+    if (!element || !className) {
+        return true;
+    }
+
+    if (nameIndex == -1) {
+        classString += ' ' + className;
+    } else {
+        classString = classString.substr(0, nameIndex) + classString.substr(nameIndex+className.length);
```
Comment (sergii-kulakovskyi) [inline djangocms_admin_style/static/js/base-admin.js]: `nameIndex+className.length` needs space around `+`

File: djangocms_admin_style/static/js/base-admin.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
+function toggleClass (element, className) {
+
+    var classString = element.className,
+        nameIndex = classString.indexOf(className);
```
Comment (dhorelik) [inline djangocms_admin_style/static/js/base-admin.js]: `var nameIndex = ..`. We usually start every new variable with a `var`, this convention makes things easier sometimes

File: djangocms_admin_style/static/js/base-admin.js [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+var html = document.getElementsByTagName('html')[0];
+
+
+menuItem.addEventListener('click', function (event) {
+    event = event || window.event;
+    event.preventDefault();
+
+    toggleClass(submenu, 'submenu-open');
+    toggleClass(menuItem, 'menu-item-open');
+
```
Comment (dhorelik) [inline djangocms_admin_style/static/js/base-admin.js]: unnecessary empty line

**Comment by dhorelik** [comment]
apart from the minor things, lgtm

**Comment by dhorelik** [comment]
:+1:

**Pull request merged**